src/views/Analysis/__snapshots__/AnalysisGame.test.tsx.snap

```diff
@@ -1,1209 +1,22 @@
-        "children": <div
-          className="analysisLayout"
->
-          <div
-                    className="moveList"
-                    id="moveList"
-          />
-          <div
-                    className="historyButtons"
-          />
-          <div
-                    className="analysisGame"
-          />
-</div>,
-        "className": "center",
-      },
-      "ref": null,
-      "rendered": Object {
-        "instance": null,
-        "key": undefined,
-        "nodeType": "host",
-        "props": Object {
-          "children": Array [
-            <div
-              className="moveList"
-              id="moveList"
-/>,
-            <div
-              className="historyButtons"
-/>,
-            <div
-              className="analysisGame"
-/>,
-          ],
-          "className": "analysisLayout",
-        },
-        "ref": null,
-        "rendered": Array [
-          Object {
-            "instance": null,
-            "key": undefined,
-            "nodeType": "host",
-            "props": Object {
-              "children": undefined,
-              "className": "moveList",
-              "id": "moveList",
-            },
-            "ref": null,
-            "rendered": null,
-            "type": "div",
-          },
-          Object {
-            "instance": null,
-            "key": undefined,
-            "nodeType": "host",
-            "props": Object {
-              "children": undefined,
-              "className": "historyButtons",
-            },
-            "ref": null,
-            "rendered": null,
-            "type": "div",
-          },
-          Object {
-            "instance": null,
-            "key": undefined,
-            "nodeType": "host",
-            "props": Object {
-              "children": undefined,
-              "className": "analysisGame",
-            },
-            "ref": null,
-            "rendered": null,
-            "type": "div",
-          },
-        ],
-        "type": "div",
-      },
-      "type": "div",
-    },
-  ],
-  Symbol(enzyme.__options__): Object {
-    "adapter": ReactSixteenAdapter {
-      "options": Object {
-        "enableComponentDidUpdateOnSetState": true,
-        "legacyContextMode": "parent",
-        "lifecycles": Object {
-          "componentDidUpdate": Object {
-            "onSetState": true,
-          },
-          "getChildContext": Object {
-            "calledByRenderer": false,
-          },
-          "getDerivedStateFromError": true,
-          "getDerivedStateFromProps": Object {
-            "hasShouldComponentUpdateBug": false,
-          },
-          "getSnapshotBeforeUpdate": true,
-          "setState": Object {
-            "skipsComponentDidUpdateOnNullish": true,
-          },
-        },
-      },
-    },
-  },
-  Symbol(enzyme.__childContext__): null,
-}
+  <div
+    className="analysisLayout"
+  >
+    <div
+      className="moveList"
+      id="moveList"
+    />
+    <div
+      className="historyButtons"
```

Comment:
This seems to have been an issue in the old snapshot as well but shouldn't our custom Components also be represented in this snapshot as a children of these divs? (they are directly used in the AnalysisGame component)

Reply:
Ah I figured it out. It doesn't have the moves so it doesn't render them.
In #169 I indirectly addressed this by decoupling the actual Analysis Display from the Routing / loading stuff.